.kitchen.dokken.yml

```diff
@@ -89,7 +89,11 @@ suites:
       - name: 2.1.6
         environment:
           CONFIGURE_OPTS: --disable-install-rdoc
-      - name: 2.2.2
+      - name: 2.4.1
```

Comment:
Since rbenv compiles rubies I kinda question if we really need to compile two versions of Ruby within the tests. But I guess it's being done for a reason or some sort of regression test.